package.json

```diff
@@ -117,13 +118,16 @@
     "rollup-plugin-terser": "^5.0.0",
     "sass-loader": "^8.0.0",
     "semver": "^7.3.2",
-    "storybook-chromatic": "^3.5.5",
+    "storybook-chromatic": "^4.0.2",
     "storybook-readme": "^5.0.9",
     "string-replace-loader": "2.3.0",
     "style-loader": "^1.0.0",
     "ts-loader": "^6.2.0",
     "typescript": "^3.8.3"
   },
+  "resolutions": {
+    "@babel/plugin-proposal-class-properties": "7.13.0"
```

Comment:
This is required for Storybook to start until https://github.com/storybookjs/storybook/issues/14197 is addressed.